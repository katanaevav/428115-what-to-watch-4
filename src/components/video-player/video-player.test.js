import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const SmallVideoPlayerSize = {
  HEIGHT: 175,
  WIDTH: 280,
};

const playerState = {
  IS_PLAYING: true,
  IS_PAUSED: false,
  VOLUME: 0.0,
};

const Movie = {
  smallPoster: `img/aviator.jpg`,
  preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
};

it(`Render video player`, () => {
  const tree = renderer.create(
      <VideoPlayer
        playerHeight = {SmallVideoPlayerSize.HEIGHT}
        playerWidth={SmallVideoPlayerSize.WIDTH}
        poster = {Movie.smallPoster}
        preview = {Movie.preview}
        volume = {playerState.VOLUME}
        isPlaying = {playerState.IS_PLAYING}
        isPaused = {playerState.IS_PAUSED}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
