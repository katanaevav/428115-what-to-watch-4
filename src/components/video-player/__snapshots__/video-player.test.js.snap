// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render video player 1`] = `
<video
  height={175}
  poster="img/aviator.jpg"
  preload="metadata"
  width={280}
/>
`;
